scanners/subfinder/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: subfinder
+description: A Helm chart for the subfinder security Scanner that integrates with the secureCodeBox.
+
+type: application
+# version - gets automatically set to the secureCodeBox release version when the helm charts gets published
+version: v3.1.0-alpha1
+appVersion: "v2.7.0"
+kubeVersion: ">=v1.11.0-0"
+annotations:
+  #versionApi: https://api.github.com/repos/REPONAME/REPO/releases/latest
```

Comment:
would be good to have this set.
This field is used by our scb bot to check daily for new releases and automatically updating the scanner.
check some other scanners, e.g. nuclei for the format